osmosis-1/osmosis.zone_assets.json

```diff
@@ -4112,6 +4112,19 @@
         }
       ],
       "_comment": "Mande Network $MAND"
+    },
+    {
+      "chain_name": "neutaro",
+      "base_denom": "ibc/DAED51CBD967A3BE0C467687970AFD97B202AFE4A1718B36936F49178AFE0133",
```

Comment:
This asset isn't registered at the Chain Registry. What is it supposed to be?

Reply:
Hiho,
yes I was trying to add neutaro. Dumb mistake on my end. Changed it.